doc/whats_new/v1.2.rst

```diff
@@ -42,7 +42,13 @@ random sampling procedures.
   :func:`linear_model._sgd_fast._plain_sgd` which is used by :class:`linear_model.SGDRegressor`
   and :class:`linear_model.SGDClassifier`. The old condition did not disambiguate between
   training and validation set and had an effect of overscaling the error tolerance.
-  This has been fixed in :pr:`23798` by :user:`Harsh Agrawal <Harsh14901>`
+  This has been fixed in :pr:`23798` by :user:`Harsh Agrawal <Harsh14901>`.
+
+- |Fix| :class:`model_selection.GridSearchCV` and
+  :class:`model_selection.RandomizedSearchCV` will rank as wort equally
```

Comment:
"as wort equally" - what is wort? Typo?

Reply:
maybe "will rank the candidates with `np.nan` scores as tied worst performers"?